com.unity.ml-agents.extensions/Runtime/Sensors/GridSensorComponent.cs

```diff
@@ -37,23 +37,23 @@ public Vector3 CellScale
         }
 
         [HideInInspector, SerializeField]
-        internal Vector3Int m_GridNumSide = new Vector3Int(16, 1, 16);
+        internal Vector3Int m_GridNum = new Vector3Int(16, 1, 16);
```

Comment:
Could this be called GridDimensions, or GridSize? GridNum is confusing to me.

Reply:
I was thinking this as "number of grids" on each side. I'll change it to GridSize.